doc/release-notes.md

```diff
@@ -29,51 +29,49 @@ version 0.13 should not require any additional actions.
 Downgrade warning
 -----------------
 
-### Downgrade to a version < 0.13.0.0
+### Downgrade to a version < 0.14.0.0
 
-Downgrading to a version smaller than 0.13 is not supported anymore as DIP2/DIP3 has
+Downgrading to a version smaller than 0.14 is not supported anymore as DIP8 has
 activated on mainnet and testnet.
 
-### Downgrade to versions 0.13.0.0 - 0.13.3.0
+### Downgrade to versions 0.14.0.0 - 0.14.0.1
 
-Downgrading to 0.13 releases is fully supported until DIP0008 activation but is not
-recommended unless you have some serious issues with version 0.14.
+Downgrading to older 0.14 releases is fully supported but is not
+recommended unless you have some serious issues with version 0.14.0.2.
 
 Notable changes
 ===============
 
-Fixed governance votes pruning for invalid masternodes 
-------------------------------------------------------
-A community member reported a possible attack that involves DoSing masternodes to force the network
-to prune all governance votes from this masternodes. This could be used to manipulate vote outcomes.
+Performance improvements
+------------------------
+Slow startup times were observed in older versions. This was due to sub-optimal handling of old
+deterministic masternode lists which causes loading of too many lists into memory. This should be
```

Comment:
Tense change seems technically wrong
```suggestion
deterministic masternode lists which caused loading too many lists into memory. This should be
```

Reply:
This suggestion doesn't seem to flow as well as the previous... "... caused loading too many..." sounds weird. "...caused the loading of too many ..." sounds better IMO.

Reply:
agreed ```suggestion
deterministic masternode lists which caused the loading of too many lists into memory. This should be
```